src/components/Text/Text.tsx

```diff
@@ -13,7 +13,6 @@ export interface TextProps extends TextBaseProps, ColorTextBaseProps {
     style?: React.CSSProperties;
     className?: string;
     children?: React.ReactNode;
-    title?: string;
```

Comment:
This is breaking change

Reply:
I tried to describe this property and found out that title id not used in this component.
@IsaevAlexandr should we left it here?